Can't Launch more than one application

#### Issue details
Can't launch more than one application with ANGLE emulation.
(Can't even launch with RenderDoc despite no application is running)

#### Reproduction steps/code
Launch the application with ANGLE GLES20 emulation, and then launch it again.

#### Version of libGDX and/or relevant dependencies
1.10.1-SNAPSHOT

#### Stacktrace
```java
Exception in thread "main" com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load ANGLE.
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loadANGLE(Lwjgl3Application.java:102)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:137)
	at com.andedit.dungeon.lwjgl3.Lwjgl3Launcher.createApplication(Lwjgl3Launcher.java:17)
	at com.andedit.dungeon.lwjgl3.Lwjgl3Launcher.main(Lwjgl3Launcher.java:13)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loadANGLE(Lwjgl3Application.java:98)
	... 3 more
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load ANGLE shared library windows64/libEGL.dll
	at com.badlogic.gdx.backends.lwjgl3.angle.ANGLELoader.extractFile(ANGLELoader.java:93)
	at com.badlogic.gdx.backends.lwjgl3.angle.ANGLELoader.load(ANGLELoader.java:193)
	... 8 more
Caused by: java.io.FileNotFoundException: C:\Users\Owner\AppData\Local\Temp\libgdxOwner\9b0d1e9db1460d7d\libEGL.dll (The process cannot access the file because it is being used by another process)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at com.badlogic.gdx.backends.lwjgl3.angle.ANGLELoader.extractFile(ANGLELoader.java:79)
	... 9 more
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [x] Windows
- [x] Linux
- [ ] macOS (Not sure about the mac but maybe affected)

--- a/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
+++ b/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
@@ -46,8 +46,8 @@ public class ANGLELoader {
 			}
 		}
 	}
-	
-	public static boolean compare(InputStream input1, InputStream input2) throws IOException {
+
+	public static boolean compare (InputStream input1, InputStream input2) throws IOException {
 		int b = 0;
 		while ((b = input1.read()) != -1) {
 			if (b != input2.read()) {
@@ -104,13 +104,13 @@ public class ANGLELoader {
 					}
 					hasCompared = true;
 				}
-				
+
 				// Reset input stream by creating a new stream.
 				if (hasCompared) {
 					closeQuietly(inSrc);
 					inSrc = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
 				}
-				
+
 				out = new FileOutputStream(outFile);
 				byte[] buffer = new byte[4096];
 				while (true) {

--- a/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
+++ b/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
@@ -46,6 +46,20 @@ public class ANGLELoader {
 			}
 		}
 	}
+	
+	public static boolean compare(InputStream input1, InputStream input2) throws IOException {
+		int b = 0;
+		while ((b = input1.read()) != -1) {
+			if (b != input2.read()) {
+				return false;
+			}
+		}
+		if (input2.read() == -1) {
+			return true;
+		} else {
+			return false;
+		}
+	}
 
 	static String randomUUID () {
 		return new UUID(random.nextLong(), random.nextLong()).toString();
@@ -73,21 +87,42 @@ public class ANGLELoader {
 			if (!outFile.getParentFile().exists() && !outFile.getParentFile().mkdirs()) throw new GdxRuntimeException(
 				"Couldn't create ANGLE native library output directory " + outFile.getParentFile().getAbsolutePath());
 			OutputStream out = null;
-			InputStream in = null;
+			InputStream inSrc = null;
+			InputStream inFile = null;
 
 			try {
+				inSrc = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
+				boolean hasCompared = false;
+				if (!outFile.canWrite() && outFile.exists()) {
+					inFile = new BufferedInputStream(new FileInputStream(outFile));
+					try {
+						if (compare(inSrc, inFile)) {
+							return outFile;
+						}
+					} finally {
+						closeQuietly(inFile);
+					}
+					hasCompared = true;
+				}
+				
+				// Reset input stream by creating a new stream.
+				if (hasCompared) {
+					closeQuietly(inSrc);
+					inSrc = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
+				}
+				
 				out = new FileOutputStream(outFile);
-				in = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
 				byte[] buffer = new byte[4096];
 				while (true) {
-					int length = in.read(buffer);
+					int length = inSrc.read(buffer);
 					if (length == -1) break;
 					out.write(buffer, 0, length);
 				}
 				return outFile;
 			} finally {
 				closeQuietly(out);
-				closeQuietly(in);
+				closeQuietly(inSrc);
+				closeQuietly(inFile);
 			}
 		} catch (Throwable t) {
 			throw new GdxRuntimeException("Couldn't load ANGLE shared library " + sourcePath, t);

--- a/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
+++ b/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
@@ -74,7 +74,7 @@ public class ANGLELoader {
 				"Couldn't create ANGLE native library output directory " + outFile.getParentFile().getAbsolutePath());
 			OutputStream out = null;
 			InputStream in = null;
-			
+
 			if (outFile.exists()) {
 				return outFile;
 			}

--- a/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
+++ b/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
@@ -47,16 +47,6 @@ public class ANGLELoader {
 		}
 	}
 
-	public static boolean compare (InputStream input1, InputStream input2) throws IOException {
-		int b = 0;
-		while ((b = input1.read()) != -1) {
-			if (b != input2.read()) {
-				return false;
-			}
-		}
-		return input2.read() == -1;
-	}
-
 	static String randomUUID () {
 		return new UUID(random.nextLong(), random.nextLong()).toString();
 	}
@@ -83,42 +73,25 @@ public class ANGLELoader {
 			if (!outFile.getParentFile().exists() && !outFile.getParentFile().mkdirs()) throw new GdxRuntimeException(
 				"Couldn't create ANGLE native library output directory " + outFile.getParentFile().getAbsolutePath());
 			OutputStream out = null;
-			InputStream inSrc = null;
-			InputStream inFile = null;
+			InputStream in = null;
+			
+			if (outFile.exists()) {
+				return outFile;
+			}
 
 			try {
-				inSrc = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
-				boolean hasCompared = false;
-				if (!outFile.canWrite() && outFile.exists()) {
-					inFile = new BufferedInputStream(new FileInputStream(outFile));
-					try {
-						if (compare(inSrc, inFile)) {
-							return outFile;
-						}
-					} finally {
-						closeQuietly(inFile);
-					}
-					hasCompared = true;
-				}
-
-				// Reset input stream by creating a new stream.
-				if (hasCompared) {
-					closeQuietly(inSrc);
-					inSrc = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
-				}
-
 				out = new FileOutputStream(outFile);
+				in = ANGLELoader.class.getResourceAsStream("/" + sourcePath);
 				byte[] buffer = new byte[4096];
 				while (true) {
-					int length = inSrc.read(buffer);
+					int length = in.read(buffer);
 					if (length == -1) break;
 					out.write(buffer, 0, length);
 				}
 				return outFile;
 			} finally {
 				closeQuietly(out);
-				closeQuietly(inSrc);
-				closeQuietly(inFile);
+				closeQuietly(in);
 			}
 		} catch (Throwable t) {
 			throw new GdxRuntimeException("Couldn't load ANGLE shared library " + sourcePath, t);

--- a/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
+++ b/extensions/gdx-lwjgl3-angle/src/com/badlogic/gdx/backends/lwjgl3/angle/ANGLELoader.java
@@ -54,11 +54,7 @@ public class ANGLELoader {
 				return false;
 			}
 		}
-		if (input2.read() == -1) {
-			return true;
-		} else {
-			return false;
-		}
+		return input2.read() == -1;
 	}
 
 	static String randomUUID () {

